### PR TITLE
Reduce tipping fee to 3%

### PR DESCRIPTION
**Pull request by manveerxyz**
Created a new `SupportFee` object.

Migration should automatically add a row for the new fee.

### CODE REVIEW

File: src/purchase/views.py
Comment (manveerxyz) [inline src/purchase/views.py:1]: This file should've been removed in https://github.com/ResearchHub/researchhub-backend/pull/1408 but it wasn't. Might've been a rebase issue.

File: src/researchhub_document/migrations/0057_alter_researchhubpost_document_type_and_more.py
Comment (yattias) [inline src/researchhub_document/migrations/0057_alter_researchhubpost_document_type_and_more.py:1]: I think you have some fundraise stuff in here - which is fine. Just wanted to flag it to ensure we need this here

File: src/purchase/views/purchase_view.py [hunk 1]
```diff
@@ -89,14 +89,14 @@ def create(self, request):
...
-                total_fee, rh_fee, dao_fee, current_bounty_fee = calculate_fees(
+                total_fee, rh_fee, dao_fee, current_support_fee = calculate_support_fees(
                     decimal_amount
                 )
                 if user_balance - (decimal_amount + total_fee) < 0:
                     return Response("Insufficient Funds", status=402)
                 
                 # Deduct fees from the gross amount of the purchase.
-                deduct_fees(user, total_fee, rh_fee, dao_fee, current_bounty_fee)
+                deduct_support_fees(user, total_fee, rh_fee, dao_fee, current_support_fee)
```
Comment (yattias) [inline src/purchase/views/purchase_view.py:99]: Nice job separating out the fees from bounties and tips

**Review by yattias — APPROVED** [review]

File: src/researchhub_document/migrations/0057_alter_researchhubpost_document_type_and_more.py
Comment (manveerxyz) [reply src/researchhub_document/migrations/0057_alter_researchhubpost_document_type_and_more.py:1]: Good catch this shouldn't be here.

**Pull request merged**